src/charm.py

```diff
@@ -124,6 +125,14 @@ def _config_certificate_validity(self) -> timedelta | None:
             return None
         return validity
 
+    @property
+    def _config_certificate_limit(self) -> int | None:
```

Comment:
How can the value be None now?

Reply:
They will never be None, but as discussed `model.get_config` could return a None, this is kept for the static checks.